### ISSUE TITLE
create locale dropdown for users

### ISSUE THREAD

**Issue by vzhz** [louisotto/groovy-react#3]
Currently users are told to list <code>en_us</code> if they aren't sure of their locale, and it would be much nicer, especially as you are appealing to new content managers, to offer a locations dropdown.
I'd be happy to work on this if this is a feature you would like to see.

**Comment by louisotto** [comment]
Agreed, it makes sense to have it that way. To be honest there are a few
things that need doing (updating to the latest React, etc) but feel free to
work on it :)

**Comment by vzhz** [comment]
Thanks, I won't be able to devote the time now, but I'm making a note so I make sure to loop back to this soon.
